doc-content/enterprise-only/employee-rostering/optashift-ER-overview-source-con.adoc

```diff
@@ -10,36 +10,36 @@ While some code is shared between the client and the server, you can build and u
+The modules contain a number of files, including Java classes. This document lists all the modules, as well as the the classes and other files that contain the key information for the employee rostering calculations.
 
-* `optashift-employee-rostering-benchmark` package: contains an additional application that generates random data and benchmarks the solution.
+* `employee-rostering-benchmark` module: contains an additional application that generates random data and benchmarks the solution.
 
-* `optashift-employee-rostering-distribution` package: contains readme files.
+* `employee-rostering-distribution` module: contains readme files.
 
-* `optashift-employee-rostering-gwtui` package: contains the client application with the user interface, developed using the `gwt` toolkit.
+* `employee-rostering-docs` module: contains documentation files.
 
-* `optashift-employee-rostering-server` package: contains the server application that uses {PLANNER} to perform the rostering calculation.
-** `src/main/resources/org/optaplanner/openshift/employeerostering/server/solver/employeeRosteringScoreRules.drl` file: contains the rules for the {PLANNER} calculation. These rules are written in the Drools rules language. You can modify the rules to change the logic for employee rostering.
-** `src/main/resources/org/optaplanner/openshift/employeerostering/server/roster/rosterGenerator.java` class: generates random input data for demonstration and testing purposes. If you change the requires input data, you need to change the generator accordingly.
+* `employee-rostering-gwtui` module: contains the client application with the user interface, developed using the `gwt` toolkit.
 
-* `optashift-employee-rostering-shared` package: contains the data structures shared between the server and client applications. In particular, under `src/main/resources/org/optaplanner/openshift/employeerostering/shared`, this package includes the Java classes that define the input data for the rostering calculations:
+* `employee-rostering-server` module: contains the server application that uses {PLANNER} to perform the rostering calculation.
+** `src/main/resources/org/optaweb/employeerostering/server/solver/employeeRosteringScoreRules.drl` file: contains the rules for the {PLANNER} calculation. These rules are written in the Drools rules language. You can modify the rules to change the logic for employee rostering.
+** `src/main/java/org.optaweb.employeerostering.server.roster/rosterGenerator.java` class: generates random input data for demonstration and testing purposes. If you change the requires input data, you need to change the generator accordingly.
+
+* `employee-rostering-shared` module: contains the data structures shared between the server and client applications. In particular, under `src/main/java/org/optaweb/employeerostering/shared/*`, this module includes the Java classes that define the input data for the rostering calculations, such as:
```

Comment:
In particular,  this module includes the Java classes that define the input data for the rostering calculations under the `src/main/java/org/optaweb/employeerostering/shared/* directory:

Consider reorg. In any case, IBM doesn't let us use sentence frags.

Reply:
Tricky. Handle in BXMSDOC-3285